go.mod

```diff
@@ -36,25 +36,25 @@ require (
 	github.com/andybalholm/brotli v1.0.6
 	github.com/aquasecurity/libbpfgo v0.4.5-libbpf-1.0.1
 	github.com/armon/go-radix v1.0.0
-	github.com/aws/aws-sdk-go v1.45.25
-	github.com/aws/aws-sdk-go-v2 v1.21.2
-	github.com/aws/aws-sdk-go-v2/config v1.18.45
-	github.com/aws/aws-sdk-go-v2/credentials v1.13.43
-	github.com/aws/aws-sdk-go-v2/feature/dynamodb/attributevalue v1.10.42
-	github.com/aws/aws-sdk-go-v2/feature/ec2/imds v1.13.13
-	github.com/aws/aws-sdk-go-v2/feature/s3/manager v1.11.90
-	github.com/aws/aws-sdk-go-v2/service/athena v1.31.8
-	github.com/aws/aws-sdk-go-v2/service/dynamodb v1.22.2
-	github.com/aws/aws-sdk-go-v2/service/ec2 v1.125.0
-	github.com/aws/aws-sdk-go-v2/service/ec2instanceconnect v1.17.2
-	github.com/aws/aws-sdk-go-v2/service/ecs v1.30.3
-	github.com/aws/aws-sdk-go-v2/service/glue v1.63.0
-	github.com/aws/aws-sdk-go-v2/service/iam v1.22.7
-	github.com/aws/aws-sdk-go-v2/service/rds v1.56.0
-	github.com/aws/aws-sdk-go-v2/service/s3 v1.40.2
-	github.com/aws/aws-sdk-go-v2/service/sns v1.22.2
-	github.com/aws/aws-sdk-go-v2/service/sqs v1.24.7
-	github.com/aws/aws-sdk-go-v2/service/sts v1.23.2
+	github.com/aws/aws-sdk-go v1.45.6
```

Comment:
looks like this is downgraded I'm assuming unintentionally? let's make sure to get rid of these changes

Reply:
Reverted in [3038ab8](https://github.com/gravitational/teleport/pull/33272/commits/3038ab842e2a9891a454ea1a59e9c04ffa23f8f3)